src/app/static/src/app/store/modelOptions/mutations.ts

```diff
@@ -17,6 +17,12 @@ export const mutations: MutationTree<ModelOptionsState> & MetadataMutations = {
     },
 
     update(state: ModelOptionsState, payload: DynamicFormMeta) {
-        state.optionsFormMeta = payload;
+        const allControlSectionLabels = payload.controlSections.map(c => c.label);
```

Comment:
The tests should make clear what's going on here: https://github.com/mrc-ide/hint/pull/131/files#diff-97445ba69387f1718e7c5f330133e49d